core/src/cosmos/CosmosLikeAddress.cpp

```diff
@@ -81,7 +81,7 @@ std::string CosmosLikeAddress::toBech32()
 
 std::experimental::optional<std::string> CosmosLikeAddress::getDerivationPath()
 {
-    return {};  // _derivationPath.toOptional();
+    return _derivationPath.toOptional();
```

Comment:
The old line of code do the exact same thing, right ? What is the rational behind this change ?

Reply:
_derivationPath is constructed to be `Some("")` instead of `None` by the Keychain, so the change here is to make that upstream change visible in the interface

Reply:
>  instead of ~~None~~

instead of the old "full Path" that we had to override here with a `None` to prevent bad account construction in Ledger Live*